Examples/Scripts/Python/itk_seeding.py

```diff
@@ -128,130 +215,186 @@ def runITkSeeding(field, inputSPs, outputDir, inputSpacePointsType, s=None):
+        deltaRMiddleSPRange=(deltaRMiddleMinSPRange, deltaRMiddleMaxSPRange),
+        collisionRegion=(collisionRegionMin, collisionRegionMax),
+        r=(None, rMaxSeedFinderConfig),
+        z=(zMin, zMax),
+        beamPos=beamPos,
     )
-
-    seedFilterConfig = acts.SeedFilterConfig(
-        maxSeedsPerSpM=seedFinderConfig.maxSeedsPerSpM,
-        deltaRMin=seedFinderConfig.deltaRMin,
+    seedFilterConfigArg = SeedFilterConfigArg(
         impactWeightFactor=impactWeightFactor,
-        compatSeedWeight=100,
+        compatSeedWeight=compatSeedWeight,
         compatSeedLimit=compatSeedLimit,
         numSeedIncrement=numSeedIncrement,
         seedWeightIncrement=seedWeightIncrement,
-        seedConfirmation=seedConfirmationFilter,
-        centralSeedConfirmationRange=seedFinderConfig.centralSeedConfirmationRange,
-        forwardSeedConfirmationRange=seedFinderConfig.forwardSeedConfirmationRange,
-        curvatureSortingInFilter=True,
+        seedConfirmation=seedConfirmation,
+        curvatureSortingInFilter=curvatureSortingInFilter,
         maxSeedsPerSpMConf=maxSeedsPerSpMConf,
         maxQualitySeedsPerSpMConf=maxQualitySeedsPerSpMConf,
         useDeltaRorTopRadius=useDeltaRorTopRadius,
     )
+    spacePointGridConfigArg = SpacePointGridConfigArg(
+        rMax=rMaxGridConfig,
+        zBinEdges=zBinEdges,
+        phiBinDeflectionCoverage=phiBinDeflectionCoverage,
+        phi=(phiMin, phiMax),
+        impactMax=impactMax,
+    )
+    seedingAlgorithmConfigArg = SeedingAlgorithmConfigArg(
+        allowSeparateRMax=allowSeparateRMax,
+        zBinNeighborsTop=zBinNeighborsTop,
+        zBinNeighborsBottom=zBinNeighborsBottom,
+        numPhiNeighbors=numPhiNeighbors,
+    )
+
+    return (
+        seedfinderConfigArg,
+        seedFilterConfigArg,
+        spacePointGridConfigArg,
+        seedingAlgorithmConfigArg,
+    )
+
+
+def addITkSeedingCsv(
+    s,
+    inputSPs,
+    seedfinderConfigArg: SeedfinderConfigArg = SeedfinderConfigArg(),
+    seedFilterConfigArg: SeedFilterConfigArg = SeedFilterConfigArg(),
+    spacePointGridConfigArg: SpacePointGridConfigArg = SpacePointGridConfigArg(),
+    seedingAlgorithmConfigArg: SeedingAlgorithmConfigArg = SeedingAlgorithmConfigArg(),
+):
+
+    seedFinderConfig = acts.SeedfinderConfig(
+        **acts.examples.defaultKWArgs(
+            rMin=seedfinderConfigArg.r[0],
+            rMax=seedfinderConfigArg.r[1],
+            deltaRMin=seedfinderConfigArg.deltaR[0],
+            deltaRMax=seedfinderConfigArg.deltaR[1],
+            deltaRMinTopSP=seedfinderConfigArg.deltaRTopSP[0],
+            deltaRMinBottomSP=seedfinderConfigArg.deltaRBottomSP[0],
+            deltaRMaxTopSP=seedfinderConfigArg.deltaRTopSP[1],
+            deltaRMiddleMinSPRange=seedfinderConfigArg.deltaRMiddleSPRange[0],
+            deltaRMiddleMaxSPRange=seedfinderConfigArg.deltaRMiddleSPRange[1],
+            deltaRMaxBottomSP=seedfinderConfigArg.deltaRBottomSP[1],
```

Comment:
can you reorder these, such that deltaRMinTopSP and deltaRMaxTopSP (same for bottom) are below each other? I think it makes for easier reading

Reply:
done!